test/scenarios/17. settings/general/change-language.csv

```diff
@@ -0,0 +1,15 @@
+Verify that the application handles different local times appropriately
+
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots).,,General settings page appears,
+4,Check the default language selected in the drop down For example :- local time in the Mac machine is set to 3:30 pm EST,,Default selected language should be English,
```

Comment:
Question: is 3:30 pm persistent for test case? Maybe i'm missing something but i didn't see any configuration

Reply:
Its a manual test case and an example for the local time could be anything and the drop down option should reflect the region - language. 
Also, could change the local time manually and see the drop down language change happens.

Reply:
clear, thank you !